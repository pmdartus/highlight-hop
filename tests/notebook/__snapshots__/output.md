# Test Book

By: _Test Author_

## Section 1 - Basic

### Highlight (Page 5, Location 10) - Chapter 1

> This is a highlighted text

**Note:** This is a note

---

### Note (Page 10, Location 20) - Chapter 1

**Note:** This is a note only

---

## Section 2 - Edge case

### Highlight (Page 15, Location 30) - Chapter 2

> With line breaks<br>
> and &quot;quotes&quot;

---

### Highlight (Page 15, Location 30) - Chapter 2

> With UTF-8 characters ðŸš€ and HTML tags &lt;br&gt; and &lt;b&gt;bold&lt;/b&gt;

---

### Highlight (Location 30)

> Missing page

---

### Highlight (Page 15, Location 30)

> Missing section and chapter

---
