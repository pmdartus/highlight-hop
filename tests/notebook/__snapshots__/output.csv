Type,Location,Page,Section,Chapter,Quote,Color,Note
Highlight,10,5,Section 1 - Basic,Chapter 1,This is a highlighted text,yellow,This is a note
Note,20,10,Section 1 - Basic,Chapter 1,,,This is a note only
Highlight,30,15,Section 2 - Edge case,Chapter 2,"With line breaks\nand ""quotes""",yellow,
Highlight,30,15,Section 2 - Edge case,Chapter 2,With UTF-8 characters ðŸš€ and HTML tags <br> and <b>bold</b>,yellow,
Highlight,30,,,,Missing page,yellow,
Highlight,30,15,,,Missing section and chapter,yellow,